README.md

```diff
@@ -148,8 +148,8 @@ aws secretsmanager get-secret-value --secret-id mobile-app-android-upload-key-pa
 
 ### Production Deploys
 
-Pushing a new tag will automatically deploy to the internal testing group (on iOS), and then release notes can be added and the build can be more widely published manually.
+Pushing a new tag with the `ios-` prefix will automatically deploy to the iOS internal testing group, and then release notes can be added and the build can be more widely published manually.
```

Comment:
I have not yet moved the Xcode Cloud workflow to require this prefix on iOS; I think I'll do that as I merge this PR.